frontend/src/components/TodoList.js

```diff
@@ -31,47 +38,95 @@ const TodoList = () => {
   const audioClickRef = useRef(new Audio(clickSound));
   const audioAddRef = useRef(new Audio(addSound));
 
-  const addTodo = (newTodo) => {
-    setTodos([...todos, { text: newTodo, completed: false }]);
+  useEffect(() => {
+    const fetchTodos = async () => {
```

Comment:
Eu usaria o nome `fetchToDos`, para refletir que `To Do` são 2 palavras.
O mesm ovale para as demais utilizações.